CodeSniffer/Standards/Generic/Sniffs/Commenting/TodoSniff.php

```diff
@@ -70,7 +70,7 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
             // Clear whitespace and some common characters not required at
             // the end of a to-do message to make the warning more informative.
             $type        = 'CommentFound';
-            $todoMessage = trim($matches[1]);
+            $todoMessage = trim(@$matches[1]);
```

Comment:
Hiding errors is bad practice especially within a library, that enforces good practice.

I recommend doing proper array key existence check.